cookbook/test/integration/remotezk/serverspec/default_spec.rb

```diff
@@ -0,0 +1,15 @@
+# Encoding: utf-8
+require 'spec_helper'
+
+describe 'Services' do
+  describe port(8882) do
+    it { should be_listening.with('tcp') }
+  end
+  describe file('/etc/baragon/agent.yml') do
+    it { should be_file }
+    its(:content) do
+      should match('zookeeper-1.vagrantup.com:2181,' \
+                   'zookeeper-2.vagrantup.com:2181')
+    end
```

Comment:
Similarly, might be good to make additional assertions re: the `baragon-server`, and various other things that should be enabled/running/present/configured a certain way, based on the attributes & templates.